src/test/pybind/test_rados.py

```diff
@@ -1291,3 +1295,74 @@ def test(self):
             assert_raises(NotConnected, watch1.check)
 
             assert_raises(ObjectNotFound, self.ioctx.notify, self.OID, 'test')
+
+    def make_callback_reply(self):
+        def callback(notify_id, notifier_id, watch_id, data):
+            with self.lock:
+                return data
+        return callback
+
+    def notify_callback(self, _, r, ack_list, timeout_list):
+        assert(r == 0)
+        print(f'ack_list {ack_list}')
+        with self.lock:
+            for ack in ack_list:
```

Comment:
```suggestion
            for ack_id, ack_cookie, ack_data in ack_list:
```

Reply:
dropped ack_cookie since it's not used (replaces with '_')